tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -125,6 +125,94 @@
         "connect_timeout": "{{ .connect_timeout }}",
         "lb_policy": "ROUND_ROBIN",
         {{ if eq .config.ControlPlaneAuthPolicy 1 }}
+        {{ if .sds_uds_path }}
+        "tls_context": {
+          "common_tls_context": {
+            "alpn_protocols": [
+              "h2"
+            ],
+            "tls_certificate_sds_secret_configs":[
+              {
+                "name":"default",
+                "sds_config":{
+                  "api_config_source":{
+                    "api_type":"GRPC",
+                    "grpc_services":[
+                      {
+                        "google_grpc":{
+                          "target_uri": "{{ .sds_uds_path }}",
+                          "channel_credentials":{
+                            "local_credentials":{
+                            }
+                          },
+                          "call_credentials":[
+                            {
+                              "from_plugin":{
+                                "name":"envoy.grpc_credentials.file_based_metadata",
+                                "config":{
+                                  "secret_data":{
+                                    "filename": "{{ .sds_token_path }}"
+                                  },
+                                  "header_key":"istio_sds_credentials_header-bin"
+                                }
+                              }
+                            }
+                          ],
+                          "credentials_factory_name":"envoy.grpc_credentials.file_based_metadata",
+                          "stat_prefix":"sdsstat"
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            ],
+            "combined_validation_context":{
+              "default_validation_context":{
+                "verify_subject_alt_name":[
+                  {{- range $a, $s := .pilot_SAN }}
```

Comment:
This change implies there are multiple sans.  Is that the case?

Reply:
I think this is the case. This is what we do today with file-mounted key/cert approach.

Without this SDS config, the bootstrap config contains file-mounted key/cert and ca cert. That block of config starts from line #221. The set up for SAN here is consistent with file-mounted approach in line #236

Reply:
I did some similar work and there was alot of review commentary that `SAN` should be renamed `SANs` in pilot's metadata.  I am not clear how feasible this is or was.

Just a note for the future.